rir/src/interpreter/interp.cpp

```diff
@@ -1644,7 +1660,7 @@ SEXP evalRirCode(Code* c, InterpreterInstance* ctx, SEXP env,
                     }
                 }
             }
-
+            memset(&bindingCache, 0, sizeof(bindingCache));
```

Comment:
would be nice if we only zero affected bindings

Reply:
Yes, but far from straightforward...